cli/README.md

```diff
@@ -0,0 +1,185 @@
+# maci-cli
+
+## Subcommands
+
+| Role | Action | Subcommand |
+|-|-|-|
+| User | Generate MACI keypair | `genMaciKeypair` |
+| User | Generate MACI public key | `genMaciPubkey` |
+| Coordinator | Create election | `create `|
+| User | Sign up | `signup` |
+| User | Change key / vote | `publish` |
+| Coordinator | Process one batch or all remaining batches of messages | `process` |
+| Coordinator | Tally one batch or all remaining batches of state leaves | `tally` |
+
+## Public and private key format
+
+MACI uses private keys in the BabyJub field for operations which occur within
+zk-SNARKs, such as decrypting messages or signing commands. As MACI is deployed
+on Ethereum, we seek to avoid confusing BabyJub private keys with Ethereum
+private keys. To that end, users should pass serialized formats of public and
+private keys to this CLI. We use `maci-domainobj`'s `PrivKey.serialize` and
+`PubKey.serialize` functions to do so. 
+
+Examples of serialized public and private keys:
+
+```
+Private key: macisk.e11e7e8870b6b3472e04a1ed703306ffbc9fd4ae78eff0c8b6cb552d467ef3f
+Public key:  macipk.483e37f5f4a11713d1a9c9df79fc7195f0e1e303e1d3bb823822e1930e1b2aa7
+```
+
+### Coordinator: Create election
+
+This command deploys an instance of a MACI contract.
+
+Fields that the coordinator has to set:
+
+`maci-cli create <options>`
+
+Example usage:
+
+```
+$ node build/index.js create -sk macisk.23d007423d56475d7e39dcd5053c5aa98f57a69ee85bc7813ccbf4c5e688307  -d 0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3 -u 32768 -m 32768
```

Comment:
Should we remove the `-u 32768 -m 32768` here as it currently not supported?

Reply:
Done :)